### PR TITLE
May 2023 release notes

### PR DESCRIPTION
**Pull request by can-memfault**

### CODE REVIEW

**Comment by can-memfault** [comment]
@HBehrens I opened this as a draft PR since I want to change the gif of the highlights section. Do you mind taking a look at the content nevertheless, so we can finally wrap this up fast? Thanks <3

**Comment by can-memfault** [comment]
Updated the gif

**Review by b1t3s1z3 — APPROVED** [review]
This is a nice looking write-up. I'm a little unclear as to whether the Android system property updates are the only real changes here. If so, might be worth moving that mention to the top of the doc and then going into the summary of what the Timezone selector is for. But I'm likely missing plenty of context here, so feel free to ignore my suggestion.

File: blog/2023-05-31.mdx [hunk 1]
```diff
@@ -0,0 +1,112 @@
+---
+title: May 2023
+---
+
+## Simplified Debugging with Device Timezone in Memfault
+
+The Timezone selector allow users to adjust the timezone within which all
```
Comment (HBehrens) [inline blog/2023-05-31.mdx:7]: really a bummer we don't have a section for this :( I updated MFLT-10509 to updates this paragraph once done.

File: blog/2023-05-31.mdx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+Timezone selection within Memfault is designed to make it easier for
+international teams working with globally deployed devices to contextualize,
+coordinate and collaborate. Whether you want to standardize working within the
+“Universal” timezone across your team, or you want to use the “Custom” or
+“Device” options to adjust your session to the specific local timezone of a
+device you are debugging, having this type of control over context can be
+powerful.
+
+To provide a specific example, let’s say you receive a report of a customer
```
Comment (HBehrens) [inline blog/2023-05-31.mdx:20]: I'd move the GIF above this example paragraph. This is how we usually structured these features: Value prop/abstract ideas, image, examples

**Review by HBehrens — APPROVED** [review]
one minor suggestion. Otherwise LGTM

**Pull request merged**